exchangelib/services.py

```diff
@@ -507,7 +507,15 @@ def _get_elements_in_response(self, response):
                         yield c
             except ErrorInvalidIdMalformedEwsLegacyIdFormat as e:
                 msg_text = get_xml_attr(msg, '{%s}MessageText' % MNS)
-                log.error('Caught ErrorInvalidIdMalformedEwsLegacyIdFormat error', error=str(e), exc_info=True, response_msg=msg_text)
+                log.error(
+                    'Caught ErrorInvalidIdMalformedEwsLegacyIdFormat'
+                    '\nerror: %s'
+                    '\nmsg_response: %s',
+                    str(e),
+                    msg_text,
+                    exc_info=True
```

Comment:
Should we embed `exc_info` as well or still keep it separate?

Reply:
@mypile the built-in logging support exc_info, we don't need to embed that into the log msg

Example output:

```
[ERROR] Caught ErrorInvalidIdMalformedEwsLegacyIdFormat
error: test
 msg_response: test
Traceback (most recent call last):
  File "/Users/pengfeiye/.pyenv/versions/cloud-core-2.7/lib/python2.7/site-packages/ptpython/repl.py", line 66, in run
    text = self.app.run(inputhook=inputhook)
  File "/Users/pengfeiye/.pyenv/versions/cloud-core-2.7/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 736, in run
    return run()
  File "/Users/pengfeiye/.pyenv/versions/cloud-core-2.7/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 710, in run
    return f.result()
  File "/Users/pengfeiye/.pyenv/versions/cloud-core-2.7/lib/python2.7/site-packages/prompt_toolkit/eventloop/future.py", line 151, in result
    raise self._exception
KeyboardInterrupt
```